Jackbox/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "Stringy",
+      "id": "577743466940071949"
+    },
+    "service": "Jackbox",
+    "description": {
+      "en": "It's easy to use this website to turn your phone, tablet or computer into a controller for one of Jackbox Games many party games. All you'll need is a room code"
+    },
+    "url": "www.jackbox.tv",
+    "version": "1.0",
+    "logo": "https://i.imgur.com/Dyoab1j.jpg",
+    "thumbnail": "https://i.imgur.com/poY2KNc.png",
+    "color": "#4254f4",
+    "tags": ["games"],
+    "category": "games",
+    "regExp": "jackbox[.]tv"
```

Comment:
This regex doesnt do anything, as its the same as the URL just without www.; just remove the www. at the url

Reply:
Done